### PR TITLE
CASMINST-5684 Updating list of issues fixed in the 1.2.2 readme file

### PR DESCRIPTION
**Pull request by djlapenta**
# Description

There was a request to review all changes that went into 1.2.2 in order to update the list of issues that are included in the 1.2.2 patch (in the readme file).

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

**Review by heemstra — APPROVED** [review]

File: upgrade/1.2.2/README.md [hunk 1]
```diff
@@ -5,21 +5,41 @@
...
 * Increases the timeout for the `etcd_database_health` check in the `ncn-healthcheck`
 * Fixes issue with `postgres` database backups that caused them to fail to restore and cleans up existing (bad) `postgres` backups on the system
 * Fixes the pod anti-affinity settings and pod disruption budget settings for `cray-dns-unbound`
+* Fixes problem with unbound forwarding to `powerDNS` in an air-gapped environment
 * Fixes a race condition between MEDS adding the initial BMC entries and Kea `dhcp-helper` logic updating IP addresses
-* Adds documentation for CSM post-install SNMP exporter settings
 * Fixes an issue where `snmp` credentials being set on leaf switches were being lost
 * Fixes an issue where `cray-hmcollector-poll` pod was not collecting river telemetry due to a check the collector does against the SMA `kafka` instance
 * Fixes CVEs in the `ims-load-artifacts` container image
+* Fixes CVE in the `oauth2-proxy` container image
```
Comment (jeremy-duckworth) [inline upgrade/1.2.2/README.md:20]: Suggest a broad statement that we fixed CVEs in container images. And then, we should compare 1.2.2 CVE population results with those of 1.2.1, once the final 1.2.2 release is generated.

File: upgrade/1.2.2/README.md [hunk 2]
```diff
@@ -5,21 +5,41 @@
...
+* Adds `opa` policies to force `keycloak` admin operations through CMN and to address `keycloak` vulnerability around `request_uri`
```
Comment (jeremy-duckworth) [inline upgrade/1.2.2/README.md:21]: Suggest adding reference to CVE-2020-10770 for Keycloak vuln that was mitigated. Also suggest splitting this line from the general policy rework towards least privilege.

File: upgrade/1.2.2/README.md [hunk 3]
```diff
@@ -5,21 +5,41 @@
...
+* Fixes issue with missing `App.version` field in `csi version` command
+* Adds capability to `capmc` to use the PATCH URI when trying to set multiple controls for Olympus hardware
+* Fixes failure in backing up `vcs` data when there are extra spaces in the pod name
 * Adds documentation to remediate security issues with NCN image access and secret exposure
+* Documented remediation for NCN Image access and secret exposure
```
Comment (jeremy-duckworth) [inline upgrade/1.2.2/README.md:26]: Suggest:

"Added procedure to remediate NCN image access and exposure to security hardening guide"

Collapsing this and the subsequent line.

File: upgrade/1.2.2/README.md (hunk 2 above)
Comment (jeremy-duckworth) [inline upgrade/1.2.2/README.md:21]: Suggest:

Limits Keycloak access (via ingress OPA policy) to well known OIDC endpoints for "Shasta" realm and only allow Keycloak administration through CMN.

**Review by jeremy-duckworth — CHANGES_REQUESTED** [review]

**Review by bklei — APPROVED** [review]

File: upgrade/1.2.2/README.md (hunk 1 above)
Comment (djlapenta) [reply upgrade/1.2.2/README.md]: updated as suggested..

File: upgrade/1.2.2/README.md (hunk 3 above)
Comment (djlapenta) [reply upgrade/1.2.2/README.md]: Changed per the suggestion..

File: upgrade/1.2.2/README.md (hunk 2 above)
Comment (djlapenta) [reply upgrade/1.2.2/README.md]: Changed as suggested..

Comment (jeremy-duckworth) [reply upgrade/1.2.2/README.md]: Thanks, the link was inserted by GitHub based on ~ markdown magic.

**Review by jeremy-duckworth — APPROVED** [review]

**Pull request merged**